splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/upgrade/SpliceCatalogUpgradeScripts.java

```diff
@@ -96,7 +96,7 @@ public void run() throws StandardException{
         NavigableSet<Splice_DD_Version> keys=scripts.navigableKeySet();
         for(Splice_DD_Version version : keys){
             if(currentVersion!=null){
-                if(ddComparator.compare(version,currentVersion)<0){
+                if(ddComparator.compare(version,currentVersion)<=0){
```

Comment:
i'm new, how is this related to the change in sys.sysstatements ?

Reply:
@martinrupp You are right, this change is not related to the update for the sys.sysstatements table. It is a fix for a minor issue with our upgrade process in general. 
Each upgrade script is associated with a version (let's call it S) which is usually the database version this script is introduced. 
In principal, if the currently installed database version (let's call it C) is already equal or above the version S, we don't need to call this upgrade script as the change should have already been applied. 
Before the fix, the upgrade script is still possible to be executed when C is the same as S.